Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,205 @@
+                print("Congratulations You've Beaten Me This Time")
+                print()
+                print("Would you like to play again?")
+                let playAgain = readLine() ?? " "
+                if playAgain.lowercased() == "yes" {
+                    isPlaying = true
+                } else {
+                    isPlaying = false
+                    print("Thanks for playing")
+                    break
+                }
+            }
+            
+        } else if !word.contains(guess) && !letterGuessed.contains(guess) && guess.count == 1 {
+            print()
+//            sleep(1)
+//            print("Not in word try again")
+            letterGuessed.insert(guess)
+            attemptsCount += 1
+            switch attemptsCount {
+            case 1:
+                print("""
+    +____________
+    |          |
+    |          O
+    |
+    |
+    |
+____|_______________________________________
+Oops, You have 5 more attempts!!
+""")
+                print()
+                print("Guessed Letters..... \(letterGuessed)")
+            case 2:
+//                sleep(1)
+                print("""
+    +____________
+    |          |
+    |          O
+    |          |
+    |
+    |
+____|_______________________________________
+Oops, You have 4 more attempts!!
+""")
+                print()
+                print("Guessed Letters..... \(letterGuessed)")
+            case 3:
+//                sleep(1)
+                print("""
+    +____________
+    |          |
+    |          O
+    |         /|
+    |
+    |
+____|_______________________________________
+Oops, You have 3 more attempts!!
+""")
+                print()
+                print("Guessed Letters..... \(letterGuessed)")
+            case 4:
+//                sleep(1)
+                print("""
+    +____________
+    |          |
+    |          O
+    |         /|\\
+    |
+    |
+____|_______________________________________
+Oops, You have 2 more attempts!!
+""")
+                print()
+                print("Guessed Letters..... \(letterGuessed)")
+            case 5:
+//                sleep(1)
+                print("""
+    +____________
+    |          |
+    |          O
+    |         /|\\
+    |         /
+    |
+____|_______________________________________
+Oops, You have 1 more attempt!!
+""")
+                print()
+                print("Guessed Letters..... \(letterGuessed)")
+            case 6:
+//                sleep(1)
+                print("""
+    +____________
+    |          |
+    |          O
+    |         /|\\
+    |         / \\
+    |
+____|_______________________________________
+Oops, You're Dead ☠️😵☠️😵☠️😵☠️!!
+""")
+                print("The correct word was \"\(word.uppercased())\"!!!")
+                print()
+                sleep(1)
+                print("Would you like to play again?")
+                let playAgain = readLine() ?? " "
+                if playAgain.lowercased() == "yes" {
```

Comment:
This code only goes into effect if the user types in "yes", but the user was not given directions that this is what they needed to type. Let's assume that the player needs to be given specific directions especially if the if/else statement will only work with a specific input.